xwiki-platform-core/xwiki-platform-tree/xwiki-platform-tree-webjar/src/main/webjar/tree.js

```diff
@@ -493,7 +493,11 @@ define(['jquery', 'JobRunner', 'jsTree', 'tree-finder'], function($, JobRunner)
       } else if (data.event && !$(data.event.target).hasClass('jstree-no-link') &&
           $(data.event.target).closest('.jstree-no-links').length === 0) {
         // The node selection was triggered by an user event and links are enabled.
-        window.location.href = selectedNode.a_attr.href;
+        if (data.event.ctrlKey === true) {
```

Comment:
maybe a quick comment saying "When pressing CTRL and clicking on a document tree entry, open the entry on a new tab"?

Reply:
Thanks Vincent, I added a comment in the latest commit indeed